module2/homework/transformContainers/uniquemap.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <deque>
+#include <list>
+#include <map>
+#include <string>
+
+std::map<int, std::string> removeDuplicateAndTranformToMap(std::list<std::string>& inputList,
+                                                           std::deque<int>& inputDeque);
+                                                           
```

Comment:
dziwnie mi to github pokazuje, sprawdź u siebie czy jest pusta linia

Reply:
na końcu pliku jest pusta linia, nie wiem czemu GH pokazuje brak pustej linii za pustą linią (?)